src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.figures;
+
+public class Square extends Figure {
+    private int side = 5;
+
```

Comment:
```suggestion
    private int side ;
    
    public Square(String color, int side) {
        super(color);
        this.side = side;
    }
```


fix all mistakes from this class in each class accordingly (Circle, Rectangle...)